Matlab/Matlab.sublime-syntax

```diff
@@ -1,36 +1,572 @@
 %YAML 1.2
 ---
-# http://www.sublimetext.com/docs/3/syntax.html
+# https://www.sublimetext.com/docs/syntax.html
+# https://www.mathworks.com/help/matlab/index.html
 name: MATLAB
+scope: source.matlab
+version: 2
+
 file_extensions:
   - matlab
```

Comment:
As `matlab` is the only associated file extension, syntax test file should probably be renamed to be opened using that syntax automatically.

Reply:
I must say that I've only seen Matlab source code with the `.m` file extension. So I wonder if we shouldn't better change `matlab` to `m` here in the syntax? I just was reluctant to do that yet, because I'm unsure whether there would be possible unwanted implications caused by that, e.g. for Objective-C users.

Reply:
I guess those implications are the reason for `*.m` not being bound. I just want the test file to open as Matlab files out of the box, instead of it being highlighted as Objective-C. It wouldn't change anything for real life use.

Reply:
[Wikipedia](https://en.wikipedia.org/wiki/List_of_filename_extensions_(M%E2%80%93R)#M) says Objective-C, Matlab, Mathematica, and a compiled Prolog variant called Mercury all share `.m`.

Shoutout to PackageDev for automatically switching the syntax of test files to the one that they're testing. 😉

Reply:
Okay. I've changed the file extension in 56bffd0.